featurebyte/api/feature.py

```diff
@@ -1090,7 +1090,7 @@ def sql(self) -> str:
         """
         pruned_graph, mapped_node = self.extract_pruned_graph_and_node()
         payload = FeatureSQL(
-            graph=pruned_graph,
+            graph=QueryGraph(**pruned_graph.dict(by_alias=True)),
             node_name=mapped_node.name,
         )
```

Comment:
Pydantic V2 throws error when passing `QueryGraphModel` object into a model that expects `QueryGraph` object.